Backend/src/routes/api.js

```diff
@@ -16,6 +28,145 @@ ROUTER.get("/api", (req, res) => {
   temp.addListener("finish", () => console.timeEnd("send http"));
 });
 
+
+
+// --------------------------------------------------------------------------------------------------------------------
+// Data recording
+// --------------------------------------------------------------------------------------------------------------------
+
+// Convert line to UTF-8 and remove return character
+function _convertLine(line) {
+	return line.toString('utf8').replace('\r', "");
+}
+
+
+const RECORDED_DATA_PATH = './recordedData/sessions/';
+const SESSIONS_LIST_PATH = './recordedData/sessionsList.bin';
+const PROCESS_SCRIPT_PATH = './src/routes/process_recorded_data.py';
+const DATA_FORMAT_PATH = './Data/sc1-data-format/format.json';
+const PROCESSED_DATA_PATH = './recordedData/processedData/';
+
+let doRecord = false; // Flag for whether we should be recording data or not
+let currentSession = "";
+
+let sessionsList = [];
+
+const broadbandLines = new nReadlines(SESSIONS_LIST_PATH);
+let line;
+let lineNumber = 1;
+
+// Getting all the created sessions from sessionsList.bin
+while (line = broadbandLines.next()) {
+  // console.log(`Line ${lineNumber} has: ${line.toString('ascii')}`);
```

Comment:
Should this still be here or do we want to remove commented out code?

Reply:
Addressed